logbook/logbook.py

```diff
@@ -120,57 +120,35 @@ def Fmessage_on_stop():
 
 
 #The second (preferred) type of logging is elisaLogbook, which sends the logs off to an external database.
+#Systems effected should be passed as a space separated list
 
-# $ curl --user fooUsr:barPass -d "author=jsmith&message=foo&run_num=1&run_type=TEST" -X POST http://localhost:5005/v1/elisaLogbook/message_on_start/
-@app.route('/v1/elisaLogbook/message_on_start/', methods=["POST"])
+# $ curl --user fooUsr:barPass -d "author=jsmith&title=foo&body=bar&command=start&systems=DAQ CRP" -X POST http://localhost:5005/v1/elisaLogbook/new_message/
+@app.route('/v1/elisaLogbook/new_message/', methods=["POST"])
 @auth.login_required
-def message_on_start():
-    logbook._start_new_message_thread()
-    
-    text = f"<p>User {request.form['author']} started run {request.form['run_num']} of type {request.form['run_type']}</p>"
-    if request.form['message'] != "":
-        text += "\n<p>"+request.form['author']+": "+request.form['message']+"</p>"
-    text += "\n<p>log automatically generated by NanoRC.</p>"
-    
-    title = f"{request.form['author']} started new run {request.form['run_num']} ({request.form['run_type']})"
-
+def new_message():
+    if request.form['body'] == "":
+         return {'success': False, 'response': "Message cannot be empty!", 'thread_id': -1}, 400
```

Comment:
Please use `Flask.jsonify`:
```
return jsonify(message, status=400, mimetype='application/json')
```

Reply:
Fixed, also got rid of the success field completely and the thread id for errors, since the client can use the status code to know how to treat the response.